### PR TITLE
docs: add tidb on hbase instruction

### PR DESCRIPTION
**Pull request by c4pt0r**

### CODE REVIEW

**Comment by c4pt0r** [comment]
@shenli  @ngaut  PTAL

**Comment by ngaut** [comment]
LGTM

**Comment by ngaut** [comment]
@shenli @siddontang PTAL

File: docs/HBASE_QUICKSTART.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+git clone https://github.com/pingcap/tidb.git $GOPATH/src/github.com/pingcap/tidb
+cd $GOPATH/src/github.com/pingcap/tidb
+make
+```
+
+Run command line interpreter:
+
+```
+make interpreter
+cd interpreter && ./interpreter -store hbase -dbpath localhost/tidb -dbname test
```
Comment (siddontang) [inline docs/HBASE_QUICKSTART.md]: should mention tso address here?

**Comment by shenli** [comment]
LGTM
I have tested it from scratch by following this doc.

**Comment by shenli** [comment]
Should we mention we need protoc version 2.7?

**Comment by c4pt0r** [comment]
No, mvn package doesn't need to compile .proto files.

**Pull request merged**